### ISSUE TITLE
Classic block:  TinyMCE tooltips are not displayed on focus

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6193]
TinyMCE displays its own tooltips for its buttons just on `mouseover`, see https://github.com/tinymce/tinymce/blob/6b1c354e63ef0c01314b44a7c7115c433a1b8fb3/src/core/src/main/js/ui/Widget.js#L46

However, in Gutenberg, all controls including the blocks formatting toolbars display their tooltips also on focus. That's for good accessibility reasons, as the accessible name of an UI control must be exposed and discoverable, at least with a tooltip, also for keyboard users.

Aside: the Classic block toolbar at the moment is not accessible with a keyboard, see #6118 / #6163 

For consistency and accessibility reasons I'd strongly recommend to make the TinyMCE tooltips appear also on focus. Not sure if this would be better handled upstream or if there's a way to override some TInyMCE behaviors. /Cc @azaozz @iseulde

**Comment by afercia** [comment]
Re-tested during today's accessibility bug-scrub and still a problem on current master (well, no surprise as it seems to be a TinyMCE thing). Ping @azaozz @iseulde 🙂

**Comment by afercia** [comment]
Looking into pending issues, just tested again and this is still a problem to address. Noting this issue was opened 11 months ago. @azaozz @iseulde do you think this should be addressed upstream or maybe there's a way to patch it? Thanks!

**Comment by joedolson** [comment]
Issue on TinyMCE: https://github.com/tinymce/tinymce/issues/6342

From a conversation with @azaozz, he's looked into this and there's no way to fix this outside of TinyMCE. TinyMCE has had this open for a few years, and has not progressed on it; they've switched to native `title` attributes in TinyMCE 6.

Unless somebody is willing to push this through in TinyMCE, I'm not sure that this is realistically going to happen.

**Comment by youknowriad** [comment]
The upstream issue has been closed as stale and given there's nothing that can be done in Gutenberg for this, I think we should just close the current issue as won't fix
